becoming_an_editor_or_admin.md

```diff
@@ -1,75 +1,63 @@
 ---
-title: Becoming an Editor or Admin
+title: Becoming an editor or admin
 layout: "default"
 ---
 
-# Administrator and editor accounts
+To publish on data.gov.uk, you must have an administrator (admin) or editor account.
+Both accounts allow you to create and edit datasets.
 
-You must have an 'administrator' or 'editor' account if you want to publish on data.gov.uk.
 
-If you have an administrator account, for your organisation you’ll be in charge of:
+## Permissions explained
 
-* appointing other users as editors and other administrators
-* updating details about your organisation on data.gov.uk
-* creating and editing data records on data.gov.uk
+Admins can also:
 
-If you’re an editor, you’ll be responsible for:
+* create and add new editors or admins
+* approve editor requests
+* manage the activity of editors
+* respond to FOI requests (unless otherwise specified by the publisher)
```

Comment:
should we specify the meaning of the FOI acronym the first time it's used on the page?

Reply:
All the guidance will be reviewed & rewritten after launch, so this version is only meant to be a quick re-write. But, it's such a small edit, I think it's worth doing. Will update.

Reply:
@gtdata https://github.com/datagovuk/guidance/edit/gh-pages/becoming_an_editor_or_admin.md